lib/secrets.rb

```diff
@@ -1,30 +1,24 @@
+# frozen_string_literal: true
+
 require 'vault'
 require 'openssl'
 require 'fileutils'
 
-# fixed in vault server 0.6.2 https://github.com/hashicorp/vault/pull/1795
```

Comment:
do you keep a changelog for this gem? also good to note dropping support for <0.6.2

Reply:
that's not in the gem, just the server

Reply:
as in, you no longer patch around old versions of the server... no?

Reply:
ah yes ... I though this was patching the gem not the server dug ... I'll bring that back, need more testing to remove that